src/bci_build/package/__init__.py

```diff
@@ -502,7 +502,7 @@ class BaseContainerImage(abc.ABC):
     #: URL to the logo of this container image.
     #: This value is added to the ``io.artifacthub.package.logo-url`` label if
     #: present
-    logo_url: str = ""
+    logo_url: str = "https://opensource.suse.com/bci/sle-bci-plain-pos.png"
```

Comment:
Do we have an svg?

Reply:
I don't unfortunately, I only have the single png (that I captured myself). I'll chase if I can find the orginal source and logos somewhere.